### PR TITLE
delete module

### PR DESCRIPTION
**Pull request by ksdfg**
- make a new `delete` module with /del and /purge
- make new wrappers in the module to check for `CanDeleteMessages` permissions, if exception for `delete` module is not added
- allow only admins to add and delete exceptions

### CODE REVIEW

**Comment by ksdfg** [comment]
@ksdfg #everyone

**Comment by gotenksIN** [comment]
@ksdfg Genius

File: telebot/modules/delete.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    @wraps(func)
+    def wrapper(update: Update, context: CallbackContext, *args, **kwargs):
+        # check if user can delete messages
+        if (
+            update.effective_chat.type != "private"
+            and update.effective_message.reply_to_message.from_user.id != update.effective_user.id
+            and update.effective_chat.id not in get_command_exception_chats("delete")
+        ):
+            user = update.effective_chat.get_member(update.effective_user.id)
+            if not user.can_delete_messages and user.status != "creator":
```
Comment (akhilnarang) [inline telebot/modules/delete.py:59]: Creator will always have delete messages permission nah?

Comment (ksdfg) [reply telebot/modules/delete.py:59]: well, I remember fetching the perms of myself on a group I created and I had no perms..... apparently the creator status is a big bypass

**Review by akhilnarang — APPROVED** [review]

**Pull request merged**